browserassets/html/traitorTips/trickster.html

```diff
@@ -0,0 +1,20 @@
+<link rel="stylesheet" type="text/css" href="{{resource("css/style.css")}}">
+<div class="traitor-tips">
+	<h1 class="center">You become a Trickster!</h1>
+	
+	<p>1. Spread mischief and distrust among the crew
+	<p>2. Possess someone and become them for a short time
+	</p>
+Abilities:
+<ul>
+<li>Choose haunt appearance: Allows you to copy the appearance of anyone, living or dead, and manifest as them when you haunt</li>
+<li>Mass whisper: Send a message to everyone around you.</li>
+<li>Mass emag: Disrupt the energetic flow of items and emag everything around you.</li>
```

Comment:
Gotta replace this with a description of creeping dread.

Reply:
Yup, donezo